src/Toolkit/Toolkit.Maui/SearchView/SearchView.cs

```diff
@@ -71,7 +71,7 @@ public SearchView()
 
         if (GetTemplateChild(nameof(PART_CancelButton)) is ImageButton cancelButton)
         {
-            cancelButton.Source = ImageSource.FromResource($"Esri.ArcGISRuntime.Toolkit.Maui.Assets.x{suffix}.png", Assembly.GetAssembly(typeof(SearchView)));
```

Comment:
Do we need to delete the image? Is it used elsewhere?

Reply:
Yea. You're right. I just removed files from resources now.